assets/js/AddNotificationButtonOpen.test.js

```diff
@@ -10,17 +10,79 @@ import AddNotificationButtonOpen  from './AddNotificationButtonOpen'
 import { mount } from 'enzyme'
 import HTML5Backend from 'react-dnd-html5-backend'
 import { DragDropContextProvider } from 'react-dnd'
+var reducer = require('./workflow-reducer.js');
 
 
 describe('AddNotificationButtonOpen ', () => {
 
-  var wrapper;  
+  var wrapper;
+  var notificationsOn;
+  var notificationsOff;
 
-  beforeEach(() => wrapper = mount(
-    <DragDropContextProvider backend={HTML5Backend}>
-      <AddNotificationButtonOpen/>
-    </DragDropContextProvider>
-  ));
+  beforeEach(() => {
```

Comment:
This looks nearly identical to the closed test. Can it be factored out somewhere? Is it generic enough to become a standard test workflow in utils? If not even just sharing between these two tests is a good idea. Cut and paste code gives me hives.